### ISSUE TITLE
[Docs] install-kubeadm update apt repository

### ISSUE THREAD

**Issue by koreanddinghwan** [kubernetes/website#43628]
**This is a Bug Report**

**Problem:**

- when i firstly install kubeadm by docs, the apt repository expired(https://packages.cloud.google.com/apt/doc/apt-key.gpg)

and i found this issue https://github.com/kubernetes/release/issues/2862

summarize upon issue,

- in recently, the google hosting repository support ends [see this](https://kubernetes.io/blog/2023/08/15/pkgs-k8s-io-introduction/#why-are-we-introducing-new-package-repositories)
- the community-owned repository introduced in [here](https://kubernetes.io/blog/2023/08/15/pkgs-k8s-io-introduction/#how-to-migrate)

**but documentation do not updated apt repositories into community-owned**  

**Proposed Solution:**

- the kubeadm guide should be updated

**Page to Update:**
https://kubernetes.io/ko/docs/setup/production-environment/tools/kubeadm/install-kubeadm/

**Optional Information**
used ubuntu-server 22.04

**Comment by koreanddinghwan** [comment]
i think this maybe 'korean language specific' right?

**Comment by dipesh-rawat** [comment]
> i think this maybe 'korean language page specific' right?

Certainly! The mentioned [page](https://kubernetes.io/ko/docs/setup/production-environment/tools/kubeadm/install-kubeadm/) is in Korean language.

/language ko

**Comment by dipesh-rawat** [comment]
I see the Korean version of the [page](https://kubernetes.io/ko/docs/setup/production-environment/tools/kubeadm/install-kubeadm/) is not up to date compared to the English [version](https://kubernetes.io/docs/setup/production-environment/tools/kubeadm/install-kubeadm/), and this could be the reason why the page still contains references to outdated legacy package repositories.

**Comment by dipesh-rawat** [comment]
/retitle [ko] "Installing kubeadm" page references deprecated legacy repositories

**Comment by koreanddinghwan** [comment]
maybe, i can do this. 
/assign

**Comment by koreanddinghwan** [comment]
- [ ]  1. fix pagination bug in https://kubernetes.io/ko/docs/setup/learning-environment/
- [ ]  2. fix instllation guide for kubeadm https://kubernetes.io/ko/docs/setup/production-environment/tools/kubeadm/install-kubeadm/
- [ ] 3. update https://kubernetes.io/ko/docs/setup/production-environment/tools/kubeadm/

**Comment by daeun-ops** [comment]
Hi @koreanddinghwan, may I take over this issue?

It looks like the update has stalled, and I’d be happy to help revise the Korean kubeadm install guide with the correct repository info.

If there’s no objection, I’d like to `/assign` myself and work on a PR.

cc: @ianychoi

**Comment by ianychoi** [comment]
@koreanddinghwan,I am unassigning you since there were no activities on this issue. I would suggest you to comment here or seek for another issue if possible. @daeun-ops Thank you for your stepping up.

/unassign @koreanddinghwan,
/assign @daeun-ops
